docker/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Copyright © 2017-2019 The OpenEBS Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# This Dockerfile builds cstor main container running zrepl from base image
+#
+
+FROM openebs/cstor-base:ci
+
+COPY entrypoint-poolimage.sh /usr/local/bin/
+
+RUN chmod +x /usr/local/bin/entrypoint-poolimage.sh
+
+ARG ARCH
+ARG DBUILD_DATE
+ARG DBUILD_REPO_URL
+ARG DBUILD_SITE_URL
+
+LABEL org.label-schema.name="cstor"
+LABEL org.label-schema.description="OpenEBS cStor"
+LABEL org.label-schema.schema-version="1.0"
```

Comment:
are there somethings that can be changed here? like name/description/version?

Reply:
Actually we moved the build process alone... So no need to change anything as of now